src/Elcodi/Bundle/CartCouponBundle/Tests/Functional/app/AppKernel.php

```diff
@@ -40,6 +40,7 @@ public function registerBundles()
             'Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle',
             'Elcodi\Bundle\FixturesBoosterBundle\ElcodiFixturesBoosterBundle',
             'Elcodi\Bundle\CartCouponBundle\ElcodiCartCouponBundle',
+            'Elcodi\Bundle\CartBundle\ElcodiCartBundle',
```

Comment:
Hmmm, this is not necessary indeed, CartCouponBundle should have CartBundle as a dependency, right?